client-portal/modules/card/containers/Form.tsx

```
@@ -22,83 +27,94 @@ function FormContainer({
+      },
     }).then(() => {
       Alert.success(`You've successfully created a ${type}`);
 
       closeModal();
     });
   };
 
-  const labels = labelsQuery?.data?.pipelineLabels || [];
+
+
+  const labels = data && data[`${type}PipelineLabels`] || [];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)